test/test_reductions.py

```diff
@@ -1228,6 +1228,12 @@ def _amax_wrapper(x, dim=None, keepdims=False):
         self._test_minmax_helper(_amin_wrapper, np.amin, device, dtype)
         self._test_minmax_helper(_amax_wrapper, np.amax, device, dtype)
 
+    @onlyNativeDeviceTypes
+    @dtypes(*complex_types())
+    def test_invalid_0dim_aminmax(self, device, dtype):
+        with self.assertRaisesRegex(RuntimeError, 'not implemented'):
+            torch.aminmax(torch.tensor(True, dtype=dtype, device=device), dim=0)
```

Comment:
```suggestion
            torch.aminmax(torch.tensor(1.0, dtype=dtype, device=device), dim=0)
```
could we use a scalar that is not a bool since bool's not the intended target dtype here?